src/store/modules/gateway/actions.ts

```diff
@@ -0,0 +1,59 @@
+import { Commit, ActionPayload } from 'vuex';
+import * as mutation from './types-mutations';
+import * as consts from '../../../consts/general';
+import {
+  GatewayState,
+  RootState,
+  AppGatewayServiceRequest,
+  AppGatewayServiceClient,
+  grpc,
+} from '../../types';
+
+
+interface ActionContext {
+  state: GatewayState;
+  rootState: RootState;
+  commit: Commit;
+  payload?: ActionPayload;
+}
+
+export const setNewClient = ({ state, commit }: ActionContext): any => {
+  const hostname: string = process.env.VUE_APP_HOST_NAME == null ? '' : process.env.VUE_APP_HOST_NAME;
```

Comment:
We need to update `VUE_APP_HOST_NAME` in our slack channel config file

Reply:
Are you using this value `http://localhost:50056`?

Reply:
Updated. Yes its the envoy proxy port